### PR TITLE
Disable output to match.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Move and modified from https://github.com/vesoft-inc/nebula-graph/pull/1196

### CODE REVIEW

File: tests/tck/features/match/Base.IntVid.feature [hunk 1]
```diff
@@ -423,6 +423,44 @@ Feature: Basic match
...
 
+  Scenario: Disabled input
+    When executing query:
+      """
+      YIELD [hash('Tim Duncan')] as id |
+      MATCH (v:player{name:"Tim Duncan"})-[e:like]->(v1) WHERE id(v) IN $-.id RETURN *
+      """
+    Then a SemanticError should be raised at runtime: Can't output to match sentence.
+    When executing query:
+      """
```
Comment (czpmango) [inline tests/tck/features/match/Base.IntVid.feature:434]: Please add all cypher clause test cases.
such as: 
```
go from ... | unwind ...
lookup on ... | with ...
```

**Comment by czpmango** [comment]
I think it's better to disable the mixing of `nGQL` and `cypher` directly.

**Comment by Shylock-Hg** [comment]
> I think it's better to disable the mixing of `nGQL` and `cypher` directly.

In fact, in our design the cypher sentence is part of the whole nGQL, they are not mutually exclusive. Match output to normal nGQL sentence is ok.

File: tests/tck/features/match/Base.IntVid.feature (hunk 1 above)
Comment (Shylock-Hg) [reply tests/tck/features/match/Base.IntVid.feature:434]: Done

**Review by czpmango — DISMISSED** [review]
LGTM.

**Review by czpmango — DISMISSED** [review]

**Comment by Shylock-Hg** [comment]
Replace by #3506

**Pull request closed without merging**